azure-pipelines.yml

```diff
@@ -285,7 +285,7 @@ jobs:
   displayName: Fast Verilated Earl Grey tests
   # Build and run fast tests on sim_verilator
   pool: ci-public
-  timeoutInMinutes: 240
+  timeoutInMinutes: 120
```

Comment:
This was 180 before your PR #17573. Are you sure we should decrease it below that, to 120?

Reply:
I think so. It was increased to 180 in 884644956582f3eb492949a172f7ff2f0436d47a but the regression that was fixing has since been resolved (by isolating the verilator tests from the SW B & T jobs). This job should take less than an hour so 2h seems a reasonable timeout.

@jprendes any additional thoughts?